ronad.gemspec

```diff
@@ -1,17 +1,17 @@
 # coding: utf-8
 lib = File.expand_path("../lib", __FILE__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
-require "danom/version"
+require "ronad/version"
 
 Gem::Specification.new do |spec|
-  spec.name          = "danom"
-  spec.version       = Danom::VERSION
+  spec.name          = "ronad"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.

[Read more about it here.](http://www.rubydoc.info/github/bbatsov/rubocop/Rubocop/Cop/Style/StringLiterals)